Week 1/Weekly Practice Problems (Week 1)/0709: To Lower Case/dominic.cpp

```diff
@@ -0,0 +1,9 @@
+class Solution {
+public:
+    string toLowerCase(string s) {
+        for (char &c : s) {
+            if (c >= 'A' && c <= 'Z') c = c - 'A' + 'a';
```

Comment:
This is a fine impl, though migh be easier to just use `tolower`

Reply:
Haha, I was actually giving some students the challenge of not using toLower :P